auth_jwt_test.go

```diff
@@ -989,3 +989,44 @@ func TestHTTPStatusMessageFunc(t *testing.T) {
 	assert.Equal(t, successMessage, successString)
 	assert.NotEqual(t, successMessage, failedString)
 }
+
+func TestSendAuthorizationBool(t *testing.T) {
+	// the middleware to test
+	authMiddleware := &GinJWTMiddleware{
+		Realm:             "test zone",
+		Key:               key,
+		Timeout:           time.Hour,
+		MaxRefresh:        time.Hour * 24,
+		Authenticator:     defaultAuthenticator,
+		SendAuthorization: true,
+		Authorizator: func(user interface{}, c *gin.Context) bool {
+			if user.(string) != "admin" {
```

Comment:
should use 'return <expr>' instead of 'if <expr> { return <bool> }; return <bool>'